Worktez/worktez
# Fix/quick notes UI

### Functionality:
Imporved Quick Notes Button
Made Create A Task button visible in dark mode

### Solution:
Made Required CSS Changes

### Risk level:

### How to test:
Check whether the QuickNotesButton is Visible and it is according to the Proposed UI.
Check whether the CreateATask button is visible in Dark Mode
